code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -40,21 +40,27 @@
 	if(istype(H))
 		var/obj/item/organ/external/affected = H.get_organ(user.zone_sel.selecting)
 		if(!affected)
-			user << "<span class='danger'>\The [H] is missing that limb!</span>"
+			to_chat(user, "<span class='danger'>\The [H] is missing that limb!</span>")
 			return
 		else if(affected.robotic >= ORGAN_ROBOT)
-			user << "<span class='danger'>You cannot inject a robotic limb.</span>"
+			to_chat(user, "<span class='danger'>You cannot inject a robotic limb.</span>")
 			return
 
+		if(H.a_intent != I_HELP)
+			to_chat(user, "<span class='notice'>[H] is resisting your attempt ot inject them with [src].</span>")

```

Comment:
"ot"

Reply:
And this is going to go "they're resisting" before they even start resisting, isn't it? Just because someone's not on help intent, that doesn't mean they're actively resisting you.